docs/sdk-and-tools/sdk-py/sdk-py-cookbook.md

```diff
@@ -3,7 +3,7 @@ id: sdk-py-cookbook
 title: Cookbook
 ---
 
-[comment]: # (mx-abstract)
+[comment]: # "mx-abstract"
```

Comment:
in the rest of the files we have `[comment]: # (mx-abstract)` and you changed `(` to `"`. expected? if so, we should update in the rest of the places as well

Reply:
The formatter was deleting the parentheses, should be fixed